todo/src/actions/actions.js

```diff
@@ -0,0 +1,9 @@
+export const ADD_TODO = "ADD_TODO";
+
+export function addToDo(newTodo) {
+    console.log(newTodo);
```

Comment:
This is really close. try returning your payload with the value and whether it's completed or not.